src/main/resources/jenkins/plugins/extracolumns/BuildDurationColumn/columnHeader.jelly

```diff
@@ -31,5 +31,8 @@ THE SOFTWARE.
         <j:when test="${it.getBuildDurationType()==1}">
             <th>${%Average Duration}</th>
         </j:when>
+        <j:when test="${it.getBuildDurationType()==2}">
+            <th title="${%Average duration in minutes (compact)}">⏳'</th>
```

Comment:
My assumption was wrong. It does not depend on the Jenkins version. It depends on your browser and font. It doesn't work at all for me in the current form. If I use the corresponding HTML entity in either hex or decimal, I can see at least a black and white(!) hour glass in Chrome, but nothing in Firefox. I don't know if there are other ways to do it, but I'd still recommend to use an image.
Which browser do you use? Is there a special reason why you want to use the unicode character?